examples/librimix/librimix.sh

```diff
@@ -11,8 +11,7 @@ DURATION=3
 [[ $(uname) == 'Darwin' ]] && nj=$(sysctl -n machdep.cpu.thread_count) || nj=$(grep -c ^processor /proc/cpuinfo)
 
 # Obtain MiniLibriMix
-[ -f ${LIBRIMIX_ROOT}/MiniLibriMix.zip ] || wget https://zenodo.org/record/3871592/files/MiniLibriMix.zip -P ${LIBRIMIX_ROOT}
-[ -d ${CORPUS_PATH} ] || unzip ${LIBRIMIX_ROOT}/MiniLibriMix.zip -d ${LIBRIMIX_ROOT}
+lhotse recipe librimix-obtain ${LIBRIMIX_ROOT}
```

Comment:
Perhaps a comment here saying what it's doing, e.g. downloading and unzipping the data?